### PR TITLE
News backend refactoring

### PR DESCRIPTION
**Pull request by Dreylak**

### CODE REVIEW

File: TheraLang.Web/Controllers/NewsController.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ public async Task<IActionResult> GetById(int id)
         }
 
         // POST: api/news
-        [Authorize]
+        [Authorize(Roles = "Admin")]
```
Comment (ivanko-yuz) [inline TheraLang.Web/Controllers/NewsController.cs]: would be good to make this a constant

File: TheraLang.Web/Validators/NewsValidators/NewsEditViewModelValidator.cs [hunk 2]
```diff
@@ -8,8 +8,8 @@ public class NewsEditViewModelValidator : AbstractValidator<NewsEditViewModel>
     {
         public NewsEditViewModelValidator()
         {
-            RuleFor(x => x.Title).NotNull().MinimumLength(3).MaximumLength(500);
-            RuleFor(x => x.Text).NotNull().MinimumLength(5).MaximumLength(50000);
+            RuleFor(x => x.Title).NotNull().MinimumLength(1).MaximumLength(500);
```
Comment (ivanko-yuz) [inline TheraLang.Web/Validators/NewsValidators/NewsEditViewModelValidator.cs]: those magic numbers

File: TheraLang.Web/Controllers/NewsController.cs (hunk 1 above)
Comment (Dreylak) [reply TheraLang.Web/Controllers/NewsController.cs]: Done

File: TheraLang.Web/Validators/NewsValidators/NewsEditViewModelValidator.cs (hunk 2 above)
Comment (Dreylak) [reply TheraLang.Web/Validators/NewsValidators/NewsEditViewModelValidator.cs]: Done

**Pull request merged**